private-templates-service/client/package.json

```diff
@@ -5,16 +5,21 @@
 	"dependencies": {
 		"@fortawesome/fontawesome-free": "^5.11.2",
 		"@microsoft/microsoft-graph-client": "^2.0.0",
+		"@types/markdown-it": "0.0.9",
```

Comment:
what does this do

Reply:
monaco-editor and markdownit are required dependencies for the Designer, the uses for these modules can be seen in Designer.tsx